data-plane/src/connection.rs

```diff
@@ -7,7 +7,7 @@ pub type Connection = tokio_vsock::VsockStream;
 #[cfg(not(feature = "enclave"))]
 pub async fn get_socket(port: u16) -> Result<Connection, tokio::io::Error> {
     Connection::connect(std::net::SocketAddr::new(
-        std::net::IpAddr::V4(std::net::Ipv4Addr::new(0, 0, 0, 0)),
+        std::net::IpAddr::V4(std::net::Ipv4Addr::new(172, 20, 0, 8)),
```

Comment:
should we make these environment variables?

Reply:
yes I'll add this in the follow up!